be/src/storage/txn_manager.cpp

```diff
@@ -165,20 +164,20 @@ Status TxnManager::commit_txn(KVStore* meta, TPartitionId partition_id, TTransac
                 if (load_info.load_id.hi() == load_id.hi() && load_info.load_id.lo() == load_id.lo() &&
                     load_info.rowset != nullptr && load_info.rowset->rowset_id() == rowset_ptr->rowset_id()) {
                     // find a rowset with same rowset id, then it means a duplicate call
-                    LOG(INFO) << "find transaction exists when add to engine."
-                              << "partition_id: " << key.first << ", transaction_id: " << key.second
-                              << ", tablet: " << tablet_info.to_string()
-                              << ", rowset_id: " << load_info.rowset->rowset_id();
+                    LOG(WARNING) << "Transaction altready exists. tablet: " << tablet_id
```

Comment:
It means a duplicate call, so i think INFO is better.

Reply:
OK